### ISSUE TITLE
switching out loss function?

### ISSUE THREAD

**Issue by sandraleebatista** [marakeby/pnet_prostate_paper#18]
How do we change the binary cross-entropy loss function to mean square error loss function? Is there documentation for this?
